resources/lang/en-US/general.php

```diff
@@ -186,6 +186,7 @@
     'last'					=> 'Last',
     'last_login'            => 'Last Login',
     'last_name'             => 'Last Name',
+    'last_name_format'		=> 'Last Name (doe@example.com)',
```

Comment:
I feel like having these in two places is kinda confusing. I'd probably opt to make a new array in the `resources/lang/en-US/admin/settings/general.php` with format translations.

Reply:
Something like (in `admin/settings/general.php`):

```php
    'email_formats' => [
        'last_name_format' => 'Last Name (doe@example.com)',
        'first_name_format' => 'First Name (jane@example.com)',
	... etc
    ],
```

and 

```php
    'username_formats' => [
        'last_name_format' => 'Last Name (doe)',
        'first_name_format' => 'First Name (jane)',
	... etc
    ],
```

just so that they're visually closer to each other, since they basically do the same general thing.

Reply:
yea, that would be better. alrighty.